contentcuration/contentcuration/frontend/channelEdit/views/TreeView/index.vue

```diff
@@ -115,14 +138,23 @@
     },
     data() {
       return {
+        drawer: {
+          maxWidth: DEFAULT_HIERARCHY_MAXWIDTH,
+          permanent: false,
+          open: false,
+        },
         loading: true,
       };
     },
     computed: {
       ...mapGetters('currentChannel', ['currentChannel', 'hasStagingTree', 'stagingId', 'rootId']),
-      ...mapGetters('contentNode', ['getContentNodeChildren', 'getContentNodeAncestors']),
-      isEmptyChannel() {
-        return !this.getContentNodeChildren(this.rootId).length;
+      ...mapGetters('contentNode', ['getContentNode', 'getContentNodeAncestors']),
+      hasTopics() {
```

Comment:
Hierarchy should only appear if topics are present in the channel, so this will prevent the panel from showing up if the channel only contains resources

Reply:
I think that this Github comment would be really useful as a code comment in this spot.

https://github.com/learningequality/studio/pull/2241#discussion_r485862101